test/message/test_runner_only_warning.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+require('../common');
+const { test } = require('node:test');
+
+test('only = true without --test-only flag', { only: true }, () => {});
```

Comment:
as you wrote, this test seems redundant (we already test `only` in https://github.com/richiemccoll/node/blob/3ee80447a2d1be090db85e8d4a3401e47d4fd717/test/message/test_runner_only_tests.js and in the other test files you have changed)

Reply:
Thanks for confirming, I've removed this file and updated the PR description.